content/blog/how-to-provision-hpe-greenlake-private-cloud-resources-from-servicenow-service-catalog.md

```diff
@@ -0,0 +1,155 @@
+    service user for ServiceNow integration](https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=request-for-subtenant-service-user-for-servicenow-integration.html).
+* Private cloud user in ServiceNow: Follow the below procedure to create a private cloud user in ServiceNow
+  * Log in to ServiceNow as admin user
+  * Create a HPE GreenLake for private cloud user by doing the following:
+    * Open the Users table and click New
+    * Assign the user the following roles:
+      * catalog_admin
+      * import_transformer
+      * itil
+      * rest_service
+      * xmodamorpheus_ca.integration (NOTE: You can only assign this role after installing the Morpheus plugin. If you are creating this user before that, you must go back and add this role after installing the plugin.)
+
+<!--EndFragment-->
+
+## Integrating private cloud on ServiceNow
+
+* From ServiceNow, go to **Morpheus Catalog** and select
+  **Properties**
+
+* Configure the following properties:
+
+  * The Morpheus Appliance endpoint: Enter the full URL to your HPE
+    GreenLake for private cloud appliance
+  * Username: Enter the name of the HPE GreenLake for private cloud
+    Service user(Received from HPE Support) that the Morpheus plugin
+    use to connect to the HPE GreenLake Private Cloud API
+  * Password: Enter the HPE GreenLake for private cloud Service user
+    password (Received from HPE Support)
+
+![](/img/figure1.png)
+
+## Integrating ServiceNow with HPE GreenLake for private cloud
+
+<!--StartFragment-->
+
+* From HPE GreenLake Central, locate the HPE GreenLake for private cloud service card and click the Launch icon to open the HPE GreenLake for private cloud dashboard
+* Navigate to Administration > Integrations
+* From the NEW INTEGRATION drop-down list, select ITSM ServiceNow
+* The NEW SERVICENOW INTEGRATION dialog box opens
+* From the dialog box, configure the ServiceNow integration parameters
+  * Name - Enter the integration name
+  * ENABLED - Select to enable consumption of this ServiceNow integration in HPE GreenLake for private cloud. The integration is enabled by default
+  * SERVICE NOW HOST-Enter the ServiceNow instance host URL (example: [https://your.instance.service-now.com](https://your.instance.service-now.com/)).
+  * USER - Enter a ServiceNow user that has below roles
+    * catalog_admin
+    * itil
+    * rest_service
+    * import_transformer
+    * xmodamorpheus_ca.integration
+  * PASSWORD -- Password of above-mentioned user
+  * Optional variables (CMDB CUSTOM MAPPING, CMDB CLASS MAPPING DEFAULT CMDB BUSINESS CLASS) not required for this use case
+* Click SAVE CHANGES
+
+The ServiceNow integration is now displayed in the list of integrations. Sample integration summary
```

Comment:
I think you mean to have "Sample integration summary" referring to the image below. I recommend that you place a carriage return prior to it and a colon after "summary" OR change the wording to "You can see a sample of the integration summary below:"

Reply:
Modified as suggested